modules/virt-aws-bm.adoc

```diff
@@ -0,0 +1,56 @@
+// Module included in the following assemblies:
+//
+// * virt/install/preparing-cluster-for-virt.adoc
+
+:_content-type: CONCEPT
+[id="virt-aws-bm_{context}"]
+= {VirtProductName} on AWS bare metal
+
+You can run {VirtProductName} on an Amazon Web Services (AWS) bare metal {product-title} cluster. Before you set up your cluster, review the following summary of supported features and limitations:
+
+Installing::
+
+* You can install the cluster by using installer-provisioned infrastructure, ensuring that you specify bare metal node instance types in the `install-config.yaml` file. For example, you can use the `c5n.metal` type value for a machine based on x86_64 architecture.
+** For more information, see the {product-title} documentation about installing on AWS.
+
+Accessing virtual machines (VMs)::
+
+* There is no change to how you access VMs by using the `virtctl` CLI tool or the {product-title} web console.
+* You can expose VMs by using a `NodePort` or `LoadBalancer` service. 
+** The load balancer approach is preferable because {product-title} automatically creates the load balancer in AWS and manages its lifecycle. A security group is also created for the load balancer, and you can use annotations to attach existing security groups. When you remove the service, {product-title} removes the load balancer and its associated resources.
+
+Networking::
+
+* You cannot use Single Root I/O Virtualization (SR-IOV) or bridge Container Network Interface (CNI) networks, including virtual LAN (VLAN). If your application requires a flat layer 2 network or control over the IP pool, consider using OVN-Kubernetes secondary overlay networks.
+
+Storage::
+--
```

Comment:
So, if you look at the output, this one renders with a different margin for the bullet. I assume because of the use of an open block. I think you are right that you need that to get your YAML in there cleanly, but wdyt about adding the `--` to the items above so everyone lines up together?
![image](https://github.com/openshift/openshift-docs/assets/61474374/78588e9b-467e-4559-9bba-a4f5ee73849e)

Reply:
ah, yes, thank you for catching that!